src/clj/rems/workflow/dynamic.clj

```diff
@@ -204,8 +211,7 @@
 
 (defmethod command-handler :application.command/accept-invitation
   [cmd application injections]
-  (or (invalid-user-error (:actor cmd) injections)
```

Comment:
right, the actor of a command can never be invalid since they're logged in?

Reply:
Yes, some role and the `getx-user-id are required by the API already. This was the only handler that had an explicit check.